packages/components/src/FilterDateRangePicker/FilterDateRangePicker.spec.tsx

```diff
@@ -35,95 +35,88 @@ const FilterDateRangePickerWrapper = ({
 
 describe("<FilterDateRangePicker />", () => {
   it("should not show the calendar initially", () => {
-    render(<FilterDateRangePickerWrapper />)
-    expect(screen.queryByRole("dialog")).not.toBeInTheDocument()
+    const { queryByRole } = render(<FilterDateRangePickerWrapper />)
+    expect(queryByRole("dialog")).not.toBeInTheDocument()
   })
 
   describe("Filter button", () => {
     it("should show the selected range in the button", () => {
-      render(
+      const { getByRole } = render(
```

Comment:
I've always opted for `screen.{selectorType}`, just kind of curious on the restructured approach here. Is is to reduce verbosity or something that contributes to the timeouts?

Reply:
It was an attempt to see if it was related (since I normally don't use `screen` and haven't had any timeouts recently.
Then it turns out it failed anyway, so I guess the performance is the same. But the changes were already done so I left it lol.